### ISSUE TITLE
Can not install on NC 25 : "404 Not Found"

### ISSUE THREAD

**Issue by tcoupin** [arawa/workspace#842]
I try to install the last v3 but the asset is missing from github release.

```
$ ./occ app:install workspace
Error: Client error: `GET https://github.com/arawa/workspace/releases/download/v3.0.1/workspace-3.0.1.tar.gz` resulted in a `404 Not Found` response:
Not Found
```

**Comment by wizdude** [comment]
i have the same issue with 26.0.2.

**Comment by wizdude** [comment]
Just a quick update. I contacted the developers and they advised there was a bug with the new 3.x release and they have pulled it. 

They have since updated the relevant metadata for Nextcloud so it doesn’t attempt to pull the new version. 

They plan to rerelease once they have fixed the relevant problems.

**Comment by zak39** [comment]
Hi :slightly_smiling_face: 

Sorry for the late reply :pray: 

Yes, we published the 3.0.2 release and it fixed a few bug that we had in the 3.0.0 and 3.0.1 releases.

I close this issue. If you find any bugs, please, create a new issue :slightly_smiling_face:
